unstructured/ingest/main.py

```diff
@@ -103,6 +103,13 @@ def run(self):
 
 
 @click.command()
+@click.option(
+    "--flatten-metadata",
+    is_flag=True,
+    default=False,
+    help="If set, flatten metadata by moving elements in the metadata field up a level "
```

Comment:
Could be reworded a touch. How about:

`Results in flattened json elements. Specifically, the metadata key values are brought to the top-level of the element, and the "metadata" key itself is removed.`